### ISSUE TITLE
reCaptcha ver. 2

### ISSUE THREAD

**Issue by dmkuznetsov** [acrobat/RecaptchaBundle#7]
Hello!

Do you have plans for upgrade your bundle up to ver. 2 (https://developers.google.com/recaptcha/)

**Comment by acrobat** [comment]
Great I didn't know about the v2, but yes I definitely want to support v2! I will try to make some time soon to integrate it!

**Comment by acrobat** [comment]
Should we still support V1 or move the whole library to V2 as it's a more "secure" way to prevent spam and bots

**Comment by dmkuznetsov** [comment]
Great! Thank you!

**Comment by acrobat** [comment]
But I'm thinking to remove the V1 support as the V2 is more safe for spam and bots. As far as I know the support drop of V1 won't bother the users using this library because the interface changes just a bit, but keys and settings will still be usable

**Comment by dmkuznetsov** [comment]
Yep, It is a good idea. Actually, people, who want to see old interface will use release 2.0 of your bundle
